decentraland/unity-explorer
# Feat: debug components command

## What does this PR change?

1. New command that allows to view components on the desired entity in realtime. You can use command `/show-entity-components scene_name entity_id` or debug panel
<img width="606" alt="image" src="https://github.com/decentraland/unity-explorer/assets/71646502/e282adfc-b516-43fa-885a-7994479f61ae">
<img width="380" alt="image" src="https://github.com/decentraland/unity-explorer/assets/71646502/36d376f5-4d36-4666-923c-0d2c392d1c91">

2. New command that clears all messages in the chat `/clear`

Technical details

- IChatHistory extracted from the chat and can be passed to debug panel to print some info to the chat
- IReadOnlySceneMapping allows to get a world by its coordinates or its name
- IWorldInfo helps to view components on the desired entity

## How to test?

1. Test the debug command as it is described in the PR
2. Try the /clear command to see that all chat commands dissapear

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
